discovery.md

```diff
@@ -224,11 +231,85 @@ The following sections define the attributes that appear in a Service entity.
   See the Primer for more information.
 
 - Constraints:
-  - REQUIRED
+  - conditionally REQUIRED. See above.
   - MUST be a valid UUID per RFC4122.
 - Examples:
   - `bf5ff5cc-d059-4c79-a89a-2513e45a1340`
 
+##### discversion
```

Comment:
What is the rationale for not going with simply `version`?

Reply:
I originally did use `version` but I wanted it to be clear that this is NOT the version of the service itself - this is the "Discovery Version". Note that we may have other versions later on... for example, we already have 'specversion" - so it might be better to not let anyone grab the generic "version" and avoid the possible confusion.